docs/user_guide/queue_config.md

```diff
@@ -259,11 +259,10 @@ The _users_ and _groups_ that can be configured can be one of two types:
 If the entry for users or groups contains more than one (1) entry it is always considered a list of either users or groups.
 The star "*" is the wildcard character and matches all users or groups.
 Duplicate entries in the lists are ignored and do not cause a parsing error.
-Specifying a star beside other list elements is not allowed.
+Specifying a star beside other list elements is not allowed. When "*" group is configured, limit should also be configured for at least one named group. Otherwise, parsing errors would be thrown.
```

Comment:
split into two points
- When "*" group --> When a wildcard groups
- configured, limit should also --> configured, a limit must be
- Otherwise, parsing errors would be thrown. --> Parsing will reject the configuration with limits  that do not follow this rule.